tools/configure

```diff
@@ -117,59 +115,6 @@ def parse_command_line(args):
 
     return opts
 
-def configure(machobj, output_dir, macros_format, compiler, mpilib, debug, sysos):
```

Comment:
create_newcase and configure now both follow the same path to create the Macros and Depends files.